pkg/core/mempool/mempool.go

```diff
@@ -142,7 +142,7 @@ func (m *Mempool) Run(ctx context.Context) {
 			select {
 			// rpcbus methods.
 			case r := <-m.sendTxChan:
-				handleRequest(r, m.processSendMempoolTxRequest, "SendTx")
+				go handleRequest(r, m.processSendMempoolTxRequest, "SendTx")
```

Comment:
Until we have a blocking call to rusk service (VerifyTransaction)  `processSendMempoolTxRequest` will not be performed concurrently.

Reply:
I don't foresee `VerifyTransaction` ever being a blocking call, as it does not activate the VM. I'm also quite positive that Rusk will be multi-threading when accepting requests.

Reply:
To what I remember @autholykos  mentioned recently that he expects latency up to 10ms (round-trip time) per VerifyTransaction call. If that the case, as you suggested,  multi-threading processing in rusk would make the trick. I'm not very certain how this would prevent  `double-spending` but let's discuss this when rusk service is on board.